content/blogposts/2022/three-years-at-sourcegraph.md

```diff
@@ -0,0 +1,418 @@
+
+Pay transparency is about the company having competitive compensation, perks, and benefits to acquire and retain talent. Hiring isn’t free, and is in fact quite expensive. Don’t hold off compensation calibration and rewards without good reason; it’s guaranteed to cost the company more, even if hiring the same exact person back.
+
+A sense of safety is built upon transparency, in an environment where everyone knows how they’re doing and what they should expect, with few surprises. It’s not about being conservative and avoiding risks, but having the ability to disagree and debate ideas based on their merits, without it becoming personal or fearing being punished for disagreeing with someone up the reporting chain. In a transparent environment, speaking up when in disagreement becomes everyone’s responsibility.
+
+Again, just a reminder: being transparent does not mean reaching a consensus for every decision.
+
+### Values are iterable
+
+“The only constant in life is change.”
+
+While living up to the values, make sure they are what you really want. [Values are iterable](https://github.com/sourcegraph/handbook/pull/2506), and as time and circumstances change, adjust wisely and accordingly.
+
+## Software engineering at Sourcegraph
+
+This is a big topic, and I neither want nor can write a book about it, but “there are a thousand Hamlets in the eyes of a thousand people” (a single text has many interpretations). I’d claim there are a thousand software engineering methodologies in the eyes of a thousand people, or in a thousand teams, or in a thousand organizations.
+
+Back in 2019, I wrote a blog post in Chinese about [what is it like to work remotely at Sourcegraph](https://unknwon.cn/2019/191116-sourcegraph-remote/), but only mentioned my feelings and observations, since I didn’t think I had enough experience to form consolidated opinions.
+
+What makes it different now? 3 years of experience is the magic number.
+
+### Organization hierarchy
+
+We just reorged (yet again within a year), and our latest [org chart](https://handbook.sourcegraph.com/team/org_chart/) is documented in the handhook. In the Engineering department, we had multiple organizations with one director and one group PM, then we had teams consisting of one EM, one PM, one product designer and multiple ICs (Note: Not all teams are fully staffed.)
+
+The product designer on my team (IAM) had been loaned to form a tiger team before the reorg. We had just lost our EM and one IC to form a new team under the Cloud department, and our PM had stepped up to be the group PM of the Admin Experience organization. What a sad story! 😅Is this the “darkest” moment I’ve experienced here? Not quite, there was a brief period where I was the only one left on the team.
+
+### Project management
+
+#### The curse of GitHub
```

Comment:
This is not related to the content, but a general bug in our blog I think, where heading 4 looks bigger than heading 3

<img width="350" alt="CleanShot 2022-07-28 at 00 48 36@2x" src="https://user-images.githubusercontent.com/2946214/181304148-7b9fd360-b43c-4fe1-8267-c8db15f1d7e7.png">